libioc/ZFS.py

```diff
@@ -74,6 +74,27 @@ def get_or_create_dataset(
 
         return self.create_dataset(dataset_name)
 
+    def get_empty_dataset(
+        self,
+        dataset_name: str
+    ) -> libzfs.ZFSDataset:
+        """
+        Create and return an empty ZFS dataset.
+
+        Unlike get_or_create_dataset this method deletes existing datasets.
```

Comment:
given how much effort this may be, and how we may want to reuse the same control repo, we might wanna actually use a persistent dataset here — ideas?

Reply:
In this case just use `ZFS.get_or_create` instead and handle clone on init and pull in case it already exists.

Reply:
pull would be handled by `r10k`.

Reply:
In this case you can remove my commit moving the method to ZFS.py - it's really only used in the ix plugin.

Reply:
done